Formula/atlantis.rb

```diff
@@ -0,0 +1,32 @@
+class Atlantis < Formula
+  desc "Terraform Pull Request Automation tool"
+  homepage "https://www.runatlantis.io"
+  url "https://github.com/runatlantis/atlantis/archive/v0.7.2.tar.gz"
+  sha256 "ecb0068f6ee1cacc4710b4f77e67b88e5d6b5d1dfae3bf6ce480980c93efa50d"
+  depends_on "go" => :build
+  depends_on "terraform"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = "src/github.com/runatlantis/atlantis"
+    build_dir = buildpath/dir
+    build_dir.install buildpath.children
+
+    cd dir do
+      system "go", "build", "-o", "atlantis"
+      bin.install "atlantis"
+    end
+  end
+
+  test do
+    system bin/"atlantis", "version"
+    port = 4141
+    loglevel = "info"
+    gh_args = "--gh-user INVALID --gh-token INVALID --gh-webhook-secret INVALID --repo-whitelist INVALID"
+    command = bin/"atlantis server --atlantis-url http://in.va.lid --port #{port} #{gh_args} --log-level #{loglevel}"
+    pid = Process.spawn(command)
+    system "sleep", "5"
+    system "curl", "-vk#", "http://localhost:#{port}/"
+    Process.kill("TERM", pid)
```

Comment:
Would it be possible to check for the existence of a log file and/or strings in the output while running the `atlantis server` command? I worry that there might be a silent failure not detected here.